package-lock.json

```diff
@@ -8350,13 +8380,13 @@
         },
         "npm-bundled": {
           "version": "1.0.6",
-          "resolved": "",
+          "resolved": false,
```

Comment:
Are you on the right npm? we use 6.9.0 on node 10.16.0 now

Reply:
Yes `10.16.0` I wonder why am getting that diff

Reply:
Strange, im on the same, but get different package lock
![image](https://user-images.githubusercontent.com/11341355/59838383-b975a700-9346-11e9-91a0-8201870603cd.png)